### DISCUSSION TITLE
"Must be confirmed" confirmation via API

### DISCUSSION THREAD

**Discussion by mikkihugo** [on-api/on-api#85] [General]
Hi,

Were trying to figure out the correct way to acknowledge an order that has "must be confirmed" (IPO Netadmin) in this case via the API.

Anyone has any good idea if this is doable?

**Comment by junetaw** [comment]
Is this still relevant @mikkihugo?
If so, please add a little more information. Order of what type? In what worfklow? Is it CO that must acknowledge or SP? As it is GlobalConnect I guess we are talkin 2.3 here.
